source/code/include/playfab/PlayFabEventPipeline.h

```diff
@@ -74,15 +78,18 @@ namespace PlayFab
         // users should not expect changes made to settings to take effect after ::Start is called unless the pipeline is destroyed and re-created
```

Comment:
These comments need to be updated to reflect the correct usage of the class (ie the difference in usage when using threads or not)